apps/rig_inbound_gateway/test/rig_web/presence/channel_test.exs

```diff
@@ -52,4 +61,39 @@ defmodule RigInboundGatewayWeb.Presence.ChannelTest do
     end
     assert capture_log(fun) =~ "unauthorized"
   end
+
+  @tag :smoke
+  test "subscribed user should receive message", %{conn: conn} do
+    {:ok, _msg} = RigOutboundGateway.Kafka.GroupSubscriber.wait_for_consumer_ready("rig", 0)
+    assert {:ok, _response, sock} = subscribe_and_join_user(
+      "testuser",
+      [@customer_role],
+      "user:testuser"
+    )
+
+    body = ~s({"user":"testuser","foo":"bar"})
+    produce_kafka_message(conn, body)
+
+    expected_event = "message"
+    expected_payload = %{"user" => "testuser", "foo" => "bar"}
+    assert_broadcast ^expected_event, ^expected_payload, 3000
+
+    leave sock
+  end
+
+  @tag :smoke
+  test "not subscribed user shouldn'\t receive message", %{conn: conn} do
```

Comment:
tab character ... typo?

Reply:
ye, that's a typo ... used to do automatic escaping of special characters from JS